src/test/components/elements/panels/SelectedEventDetails/index.test.tsx

```diff
@@ -106,6 +133,12 @@ describe("SelectedEventDetails", () => {
     expect(eventInfo.textContent).not.toContain("Private Event");
   });
 
+  it("does not show booking deadline if not present in the event details", async () => {
+    setupTest({ ...mockEvent, bookingDeadline: undefined } as IsaacEventPageDTO);
+    const eventInfo = await screen.findByTestId("event-details");
+    expect(eventInfo.textContent).not.toContain("Booking deadline");
```

Comment:
I think you've done it this way to keep it consistent with the rest of this test file, but in reality it would be more idiomatic to use `toHaveTextContent`, like this:

```typescript
expect(eventInfo).not.toHaveTextContent("Booking deadline");
```

You may as well change the rest of the assertions in this file, since it's a small and easy improvement.